src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1154,6 +1158,13 @@ MulticopterPositionControl::task_main()
 
 		poll_subscriptions();
 
+		/* get current rotation matrix and euler angles from control state quaternions */
+		math::Quaternion q_att(_ctrl_state.q[0], _ctrl_state.q[1], _ctrl_state.q[2], _ctrl_state.q[3]);
```

Comment:
Does someone in your team recall why this ended up here? Its non-obvious why a controller change would be part of the port.